# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1487116811 28800
#      Tue Feb 14 16:00:11 2017 -0800
# Branch stable
# Node ID 41a01755a7b9e7ff3fe06d93e67722f5ac5d3a57
# Parent  0032c2f7ee4eb6a23e63c5841598f19da7f7f5c2
version.m: document that "-release" returns an empty string (bug #50294)

* version.m: Document that the "-release" option returns an empty string.
Add %!tests for -description and -release.

diff --git a/scripts/miscellaneous/version.m b/scripts/miscellaneous/version.m
--- a/scripts/miscellaneous/version.m
+++ b/scripts/miscellaneous/version.m
@@ -31,17 +31,17 @@
 ## @table @asis
 ## @item @qcode{"-date"}
 ## for the release date of the running build,
 ##
 ## @item @qcode{"-description"}
 ## for a description of the release (always an empty string),
 ##
 ## @item @qcode{"-release"}
-## for the name of the running build,
+## for the name of the running build (always an empty string),
 ##
 ## @item @qcode{"-java"}
 ## for version information of the Java @nospell{VM},
 ##
 ## @item @qcode{"-fftw"}
 ## for version information for the linked @sc{fftw},
 ##
 ## @item @qcode{"-blas"}
@@ -112,15 +112,18 @@ endfunction
 
 %!test
 %! [v, d] = version ();
 %! assert (v, OCTAVE_VERSION);
 %! assert (d, __octave_config_info__ ("release_date"));
 
 %!assert (version ("-date"), __octave_config_info__ ("release_date"))
 
+%!assert (version ("-description"), "")
+%!assert (version ("-release"), "")
+
 ## Test input validation
 %!error version ("-date", "-release")
 %!error [v, d] = version ("-date")
 %!error version (1)
 %!warning <option '-blas' not implemented> version ("-blas");
 %!warning <option '-lapack' not implemented> version ("-lapack");
 %!error <invalid FEATURE> version ("-foobar")
